tests/src/test/scala/com/nvidia/spark/rapids/UrlFunctionsSuite.scala

```diff
@@ -98,6 +98,764 @@ class UrlFunctionsSuite extends SparkQueryCompareTestSuite {
     ).toDF("urls")
   }
 
+  def failedMaliciousUrlDatasets(session: SparkSession): DataFrame = {
+    // scalastyle:off line.size.limit
+    import session.sqlContext.implicits._
+    Seq[String](
+      "grbj.lyjq.org:8989/server.exe",
```

Comment:
Because many of these URLs are malicious I am not 100% sure that we want to check them in.  @sameerz are you okay with these being checked in?  I could try and go back and just get the non-malicious URLs in the data sets I found and see if that still repros the problems. Another option would be to check these in as a parquet file so at least they are not in plain text.

Reply:
Ok, I've figured out why these cases fail, I can generate some random urls and can reproduce the problems in the final version.

Reply:
>  @sameerz are you okay with these being checked in? 

No, do not check in the malicious urls.